src/sonic-config-engine/sonic_device_util.py

```diff
@@ -5,6 +5,9 @@
 import re
 from natsort import natsorted
 import glob
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicDBConfig
```

Comment:
Suggest to combine these into a single line,

```
from swsssdk import ConfigDBConnector, SonicDBConfig
```

Reply:
@jleveque  Done